Microsoft.OneWeek.Hack.Microids.MessageRouter/IoTDeviceDataEnricher.cs

```diff
@@ -16,12 +17,41 @@ public IoTDeviceGrpcDataEnricher()
             this.Client = new IoTDevice.IoTDeviceClient(channel);
         }
 
+        private static string CacheGrpcEndpoint
+        {
+            get
+            {
+                string s = System.Environment.GetEnvironmentVariable("CACHE_GRPC_ENDPOINT");
+                if (string.IsNullOrEmpty(s)) return "localhost:5000";
+                return s;
+            }
+        }
+
         private IoTDevice.IoTDeviceClient Client { get; set; }
 
         public Task<DeviceMetadata> GetMetadataAsync(string id)
         {
+            /*
+            var mre = new ManualResetEventSlim();
```

Comment:
Remove commented-out code - Git keeps this in the history, in case we ever need to get it back.